eiffel-vocabulary/EiffelActivityFinishedEvent.md

```diff
@@ -0,0 +1,40 @@
+# EiffelActivityFinishedEvent
+The EiffelActivityFinishedEvent declares that a previously queued activity (declared by [EiffelActivityQueuedEvent](./EiffelActivityQueuedEvent.md)) has finished.
```

Comment:
This indicates that one could skip the started event which is not wanted behavior since we have the dequeued event for that case. Suggest to remove "previously queued" or change to "previously started".